### DISCUSSION TITLE
Training on Loihi

### DISCUSSION THREAD

**Discussion by Rashedape** [lava-nc/lava-dl#172] [General]
Hello, I am new to Lava. Can I use the Lava-dl library for training on Loihi? I am looking for an example. Can anybody help me?

**Comment by Michaeljurado42** [comment]
I think lava-dl is just for offline training. For on chip learning you are going to want to look at local neuron learning rules I believe. There are a few tutorials on the main lava repo like [this](https://github.com/lava-nc/lava/blob/main/tutorials/in_depth/three_factor_learning/tutorial01_Reward_Modulated_STDP.ipynb).

**Reply by Rashedape** [reply]
Thank you for getting back to me so quickly.
